### PR TITLE
button created and displayed on individual job postings but not caree…

### PR DESCRIPTION
**Pull request by jstavh**
Join our team yellow written button appears in individual job postings but not on the career page. The order is now branch jason-button into-> jason-transition -> jason-career-layout -> main. Homepage does not point to the career page yet, which is next up with a nav-bar.

### CODE REVIEW

File: src/astro/src/components/Join.astro [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+import '../styles/styles.css';
+---
+
+<div>
+    <div class="mb-10">
+        <a href="mailto:hello@granola.team" class="py-4 px-8 rounded-lg text-yellow-300 no-underline text-2md
```
Comment (robinbb) [inline src/astro/src/components/Join.astro]: Change to Granola colour.

**Review by robinbb — CHANGES_REQUESTED** [review]

**Review by robinbb — APPROVED** [review]

**Review by robinbb — CHANGES_REQUESTED** [review]
Join button still appears.

**Comment by jstavh** [comment]
> Join button still appears.

I believe because we are wrapping dynamic routing [...page].astro page with <CareersLayout> <CareersLayout /> we need Join function in both. Does this make sense?

**Comment by robinbb** [comment]
Join button now appears twice on /careers page.

**Comment by jstavh** [comment]
npm run dev shows the button only once. I'm working on finding a way to get the netlify preview to match this. I added a gitignore netlify local file at the root but that did not help.

**Comment by robinbb** [comment]
I have made improvements to the build which ensure that any evidence of a prior build is removed. ('npm run clean') This is on another branch (and PR), and I have merged that branch onto this one.

Still, both buttons appear on the preview. They also appear for me when I run 'npm run dev' or 'npm run preview'. In other words, there are 2 buttons appearing due to the logic in this PR (which is therefore incorrect, and in need of correction). They may not be appearing on *your* screen due to the size of your screen. You will see them in the "view source" on your preview, likely, @jstavh.

**Pull request closed without merging**